modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -1183,7 +1181,11 @@ CrackFrontDefinition::isPointWithIndexOnIntersectingBoundary(const unsigned int
   }
   else
   {
-    //TODO: Implement for CRACK_FRONT_POINTS
+    // If the intersecting boundary option is used with crack front points, the
+    // first and last points are assumed to be on the intersecting boundaries.
+    unsigned int num_crack_front_points = getNumCrackFrontPoints();
+    if (point_index == 0 || point_index == num_crack_front_points - 1)
+      is_on_boundary = true;
   }
```

Comment:
I just slightly revised it.  Same idea, but no need to get the info from the parameters the way I did.